docs/project-structure.md

```diff
@@ -12,111 +12,90 @@ Sample applications are not covered.
 - [Desktop.UnitTests\React.Windows.Desktop.UnitTests.vcxproj](#React.Windows.Desktop.UnitTests)
 - [Folly\Folly.vcxproj](#Folly)
 - [FollyWin32\FollyWin32.vcxproj](#FollyWin32)
-- [IntegrationTestScripts\IntegrationTests.njsproj](#IntegrationTests-Node-Project)
 - [IntegrationTests\React.Windows.IntegrationTests.vcxproj](#React.Windows.IntegrationTests)
-- [JSI\Desktop\JSI.Desktop.vcxproj](#JSI.Desktop)
-- [JSI\Shared\JSI.Shared.vcxitems](#JSI.Shared)
-- [JSI\Universal\JSI.Universal.vcxproj](#JSI.Universal)
-- [JSI.Desktop.UnitTests\JSI.Desktop.UnitTests](#JSI.Desktop.UnitTests)
 - **[Microsoft.ReactNative\Microsoft.ReactNative.vcxproj](#Microsoft.ReactNative)**
 - [Microsoft.ReactNative.Cxx\Microsoft.ReactNative.Cxx.vcxitems](#Microsoft.ReactNative.Cxx)
 - [Microsoft.ReactNative.Managed\Microsoft.ReactNative.Managed.csproj](#Microsoft.ReactNative.Managed)
 - [Microsoft.ReactNative.SharedManaged\Microsoft.ReactNative.SharedManaged.projitems](#Microsoft.ReactNative.SharedManaged)
 - [ReactCommon\ReactCommon.vcxproj](#ReactCommon)
-- [Shared\Shared.vcxitems](#Shared)
+- [Shared\Shared.vcxitems](#Microsoft.ReactNative.Shared)
 
 ## Common Projects
 
 ### Common
-*Static Library*<br/>
-Functionality shared between React and JSI projects that do not depend on React or JSI functionality.
+*Static Library*\
+Functionality shared between React and JSI projects.
 
-### Shared
-*Shared Items (no build artifact)*<br/>
+### Microsoft.ReactNative.Shared
+*Shared Items (no build artifact)*\
 Holds sources common to both Windows variants, that require different build configuration
 (i.e. compiler flags, language standard).
 
 ### ReactCommon
-*Static Library*<br/>
```

Comment:
><br/> [](start = 16, length = 5)

(nit) Nice! Thanks for this clean up! Also double space should work here to denote new line.